pyrobots/src/login/Login.js

```diff
@@ -0,0 +1,62 @@
+import { useState } from 'react'
+import { setToken, fetchToken } from '../common/Auth'
+import { useNavigate } from "react-router-dom";
+import axios from 'axios';
+
+function Login() {
+  const navigate = useNavigate();
+
+  const [user, setUser] = useState({
+    email: '',
+    password: ''
+  })
+
+  const [error, setErrors] = useState('')
+
+  const handleInputChange = (event) => {
+    setUser({
+      ...user,
+      [event.target.name]: event.target.value
+    })
+  }
+
+  const login = () => {
```

Comment:
According to Santi, this function should be called handleSubmit... But this name really sounds what this function does, so is up to you if you want to change it.